src/SmartComponents/Inventory/applications/vulnerabilities/Vulnerabilities.js

```diff
@@ -3,17 +3,36 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { fetchCveListBySystem } from '../../../../redux/actions/applications';
 import VulnerabilitiesCves from './VulnerabilitiesCves';
-import { Card, CardBody } from '@patternfly/react-core';
+import { Card, CardBody, Tooltip } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
 import { createCveListBySystem } from './DataMapper';
 import { sortable, cellWidth } from '@patternfly/react-table';
 import './vulnerabilities.scss';
 
+const cvssBaseDescription = 'All CVEs use Common Vulnerability Scoring System v3 except where noted.';
+
 const header = [
     { title: '', key: 'impact', transforms: [ sortable, cellWidth(10) ]},
     { title: 'Name', key: 'synopsis', transforms: [ sortable, cellWidth(10) ]},
     { title: 'Description', key: 'description', transforms: [ cellWidth('max') ]},
-    { title: 'CVSS Base Score', key: 'cvss_score', transforms: [ sortable, cellWidth(10) ]},
-    { title: 'Publish date', key: 'public_date', transforms: [ sortable, cellWidth(10) ]}
+    {
+        title: (
+            <React.Fragment>
+                <Tooltip
+                    position="top"
+                    content={ <div>{ cvssBaseDescription }</div> }
+                >
+                    <span className="table-header-icon">
+                        <InfoCircleIcon arial-label={ cvssBaseDescription } aria-hidden="false" />
```

Comment:
Why are you using `aria-hidden: false`?

Reply:
Because the default value of `aria-hidden` attribute is `true`. And we need to get the `aria-label` readable for screen readers.

Reply:
Looking at the icons for PF, they aren't assigned a value for `aria-hidden` meaning that screen readers should pick it up because of the label.

Reply:
Unfortunately that's not correct. When I remove the `aria-hidden="true"` attribute:
`<InfoCircleIcon className={ 'table-header-icon' } arial-label={ cvssBaseDescription } />`

Patterfly adding `aria-hidden="false"`. Really:
![screenshot 2019-02-28 14 24 57](https://user-images.githubusercontent.com/1569401/53569744-2b8e6a00-3b65-11e9-9f37-6c02c7165031.png)

Reply:
Ah yes. I see now, that's because you're already describing the icon with "CVSS Base Score" if it was `aria-hidden=false`, then screen readers would read it twice.